cms/djangoapps/contentstore/views/preview.py

```diff
@@ -124,6 +125,28 @@ def user_id(self):
         return self._request.user.id
 
 
+class StudioPermissionsService(object):
```

Comment:
@dmitchell Did you see this change?

Having something in library_content_module.py "xblock.want" something in cms makes me quesy. @bradenmacdonald How would you feel about putting StudioPermissionsService in student.auth where has_studio_read_access and has_studio_write_access lives?